### ISSUE TITLE
[Bug]: Documentation bug resulting ERROR cannot find pipline image-embedding

### ISSUE THREAD

**Issue by SebLKMa** [towhee-io/towhee#2561]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

From https://docs.towhee.io/Getting%20Started/quick-start/  
`p = AutoPipes.pipeline('image-embedding')`  

will result in  
`auto_pipes.py-auto_pipes:76 - ERROR: Can not find the pipline image-embedding`

### Expected Behavior

The 'output' variable should be a vector of embeddings.

### Steps To Reproduce

```markdown
1. Follow the codes example from https://docs.towhee.io/Getting%20Started/quick-start/
2. Run in notebook or python3 (version 3.10.6 for me)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[15], line 6
      4 p = AutoPipes.pipeline('image-embedding')
      5 # Get the embedding vectors
----> 6 output = p('https://raw.githubusercontent.com/towhee-io/towhee/main/towhee_logo.png').get()
```

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93): 1.0.0rc1
- OS(Ubuntu or CentOS): Ubuntu
- CPU/Memory: Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (4 cores) / 8Gb Memory
- GPU: none
- Others:
```

### Anything else?

Following unit tests from `https://github.com/towhee-io/towhee/blob/main/tests/testcases/test_image_embedding.py`,
"text_image_embedding" instead of "mage-embedding" should be used.
```
p = AutoPipes.pipeline('text_image_embedding')
```

**Comment by shiyu22** [comment]
@SebLKMa Thanks for your issue, there is a pr#2556 to update the docs.

It is because that towhee1.0.0 is not supported to run Autopipes in Towhee Hub, but towhee1.0.0rc1 can.

**Comment by SebLKMa** [comment]
'sentence_embedding' instead of 'image_embedding'
I believe this has been fixed by [#2556](https://github.com/towhee-io/towhee/pull/2556)
